NitroxModel/Server/ServerConfig.cs

```diff
@@ -200,9 +191,6 @@ public ServerSerializerMode SerializerModeEnum
         public PlayerStatsData DefaultPlayerStats => new PlayerStatsData(oxygenSetting.Value, maxOxygenSetting.Value, healthSetting.Value, foodSetting.Value, waterSetting.Value, infectionSetting.Value);
         #endregion
 
-        public bool IsGameMode(string gamemode)
-        {
-            return GameMode == gamemode;
-        }
+        public bool IsHardcore => GameModeEnum == ServerGameMode.HARDCORE;
```

Comment:
I believe this was discussed before to have with the function as to allow for easy implementation of other gamemode checks. I would therforew suggest that this is left the same but checks the enum instead